README.md

```diff
@@ -283,6 +312,33 @@ print(json.dumps(report, indent=4))
 print(json.dumps(report["data stats"][0], indent=4))
 ```
 
+### Unstructured profiler
+In addition to structure profiler, DataProfiler provides unstructured profiling for the TextData object or string. Unstructured profile also works with list(string), pd.Series(string) or pd.DataFrame(string) given profiler_type option specified as `unstructured`. Below is an example of unstructured profile with a text file. 
+```python
+import dataprofiler as dp
+import json
+
+my_text = dp.Data('text_file.txt')
+profile = dp.Profiler(my_text)
+
+# print the report using json to prettify.
+report = profile.report(report_options={"output_format":"pretty"})
+print(json.dumps(report, indent=4))
+```
+
+Another example of unstructured profile with pd.Series of string is given as below
+```python
+import dataprofiler as dp
+import pandas as pd
+import json
+
+text_data = pd.Series(['first string', 'second string'])
```

Comment:
this will not work as `Profiler` assumes all `pd.Series` is structured.

Can you test all of the examples as well to ensure success?

Reply:
added and tested, forgot to add the option (I commented out for another example)